apps/photos/src/components/PhotoViewer/FileInfo/MapBox.tsx

```diff
@@ -0,0 +1,54 @@
+import { useEffect, useState } from 'react';
+import dynamic from 'next/dynamic';
+import { runningInBrowser } from 'utils/common';
+import 'leaflet/dist/leaflet.css';
+import 'leaflet-defaulticon-compatibility/dist/leaflet-defaulticon-compatibility.webpack.css'; // Re-uses images from ~leaflet package
+const L = runningInBrowser() ? require('leaflet') : null;
+runningInBrowser() ? require('leaflet-defaulticon-compatibility') : null;
+
+interface MapBoxProps {
+    location: { latitude: number; longitude: number };
+}
+
+const MapBox: React.FC<MapBoxProps> = ({ location }) => {
+    const [isClient, setIsClient] = useState(false);
+    const position: [number, number] = [location.latitude, location.longitude];
+
+    useEffect(() => {
+        setIsClient(true);
+    }, []);
+
+    useEffect(() => {
+        if (isClient) {
+            const mapContainer = document.getElementById('map-container');
+            if (mapContainer && !mapContainer.hasChildNodes()) {
+                const map = L.map(mapContainer).setView(position, 13);
+
+                L.tileLayer(
+                    'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png',
+                    {
+                        attribution:
+                            '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors',
+                    }
+                ).addTo(map);
+
+                L.marker(position)
+                    .addTo(map)
+                    .bindPopup('You were here.')
```

Comment:
The pop up message could be something else.. 
"Photo's location" or something else?

Reply:
Okay, done

Reply:
What is your opinion on removing the popup message entirely.

Reply:
Can remove it. wdyt @ashilkn ?